foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/synchronous.processor.impl.py.vm

```diff
@@ -42,6 +42,17 @@ class ${step.capitalizedName}(${step.capitalizedName}Base):
         #end
         #end
 
+    #if ($step.isPersistTypeRdbms())
```

Comment:
S: Rather than copying this identical snippet between both processor impl templates, can we pull it out to its own file and include by reference with a #parse call?